pages/content/amp-dev/community/roadmap.html

```diff
@@ -2,86 +2,185 @@
+              role="button"
+              tabindex="{{ loop.index }}">
+            <h4 class="ap-t-roadmap-list-item-issue-header-title">{{ issue.wg_name }}</h4>
+            <span class="ap-t-roadmap-list-item-issue-header-subtitle">Status Update {{ issue.status_update }}</span>
+            <svg class="ap-t-roadmap-list-item-issue-header-toggle"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#chevron-vertical-light"></use></svg>
           </div>
-          <h4>{{ _(column.name) }}</h4>
-          <ul>
-            {% for card in column.cards %}
-            <li class="ap-o-roadmap-tiles-item ap-m-copy active {% if card.issue.labels[0] %}{% for label in card.issue.labels %}{{ label.name | slug}} {% endfor %}{% endif %}"
-            [class]= "{% if card.issue.labels[0] %}{% for label in card.issue.labels %}roadmapfilter.activefilter == '{{ label.name | slug}}' ||{% endfor %}{% endif %} roadmapfilter.activefilter == 'all' ? 'ap-o-roadmap-tiles-item ap-m-copy active ' : 'ap-o-roadmap-tiles-item ap-m-copy '"
-            >
-              <a class="ap-o-roadmap-tiles-link" href="{{ card.issue.url }}">
-                <div class="ap-o-roadmap-tiles-title">{{ card.issue.title }}</div>
+
+          <div class="ap-t-roadmap-list-item-issue-body">
+            {% for column in issue.body|slice(2) %}
+            <div class="ap-t-roadmap-list-item-issue-body-column">
+              {% for paragraph in column %}
+              {{ paragraph|markdown|safe }}
+              {% endfor %}
+              {% if loop.last %}
+              <a href="{{ issue.html_url }}" class="ap-m-lnk" target="_blank" rel="noopener">
+                <div class="ap-a-ico ap-m-lnk-icon">
+                  {% do doc.icons.useIcon('icons/internal.svg') %}
+                  <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#internal"></use></svg>
+                </div>
+                <span class="ap-m-lnk-text">{{ _('All details on GitHub') }}</span>
               </a>
-              <div class="ap-o-roadmap-tiles-description">{{ _(card.issue.description|markdown|safe) }}</div>
-              <div class="ap-o-roadmap-tiles-categories">
-                {% if card.issue.labels[0] %}
-                  {% for label in card.issue.labels %}
-                  <button class="ap-m-filter-bubble {{ label.name | slug }} active" on="tap:AMP.setState({roadmapfilter: {activefilter: '{{ label.name | slug}}' }})">{{ _(label.name) }}</button>
-                  {% endfor %}
-                {% endif %}
-              </div>
-            </li>
+              {% endif %}
+            </div>
             {% endfor %}
-          </ul>
+            <button class="ap-t-roadmap-list-item-issue-body-expand"
+                on="tap:{{ issue.wg_slug }}-{{ issue.number }}.toggleClass(class='expanded'), {{ issue.wg_slug }}-{{ issue.number }}.scrollTo(duration=250)">
```

Comment:
Since you are reusing this action twice you could think about using [`amp-action-macro`](https://amp.dev/documentation/components/amp-action-macro/?format=websites)